src/comments.ts

```diff
@@ -90,6 +90,13 @@ export const WaitUntilMergeIsOK = (user: string, abbrOid: string, uri: string) =
 Thanks, and happy typing!`
 });
 
+export const Suggestions = (user: string) => ({
+    tag: "suggestions",
+    status: `@${user} I noticed these differences from the required form. If you can revise your changes to avoid them, so much the better! Otherwise a maintainer will take a look.
+
+If it's obvious why the differences are needed, feel free to "resolve" this review. Thanks!`,
```

Comment:
I think that it's better to tell people that they should leave a comment about why a change is needed.  It would be more useful for reviewers to see the comment and the explanation rather than having the author resolve it.  (Especially since the author's POV would usually make them want to resolve it.)

Reply:
Done :heavy_check_mark: https://github.com/DefinitelyTyped/dt-mergebot/blob/2819165e99ecdc83c166121b10e95e2ae9ad582d/src/comments.ts#L97